tests/test-components/build.rs

```diff
@@ -30,19 +35,42 @@ fn main() {
             .env("RUSTFLAGS", rustflags())
             .env("CARGO_TARGET_DIR", &out_dir);
         eprintln!("running: {cargo:?}");
-        let status = cargo.status().unwrap();
+        let status = cargo.status().expect("`cargo build` failed");
         assert!(status.success(), "{status:?}");
-        eprintln!("{status:?}");
         let const_name = to_shouty_snake_case(&package);
-        let binary_name = manifest.package.unwrap().name.replace('-', "_");
-        let wasm = out_dir
+        let package_name = manifest.package.expect("manifest has no package").name;
+        let binary_name = package_name.replace(['-', '.'], "_");
+        let wasm_path = out_dir
             .join("wasm32-wasi")
             .join("debug")
             .join(format!("{binary_name}.wasm"));
 
+        let adapter_version = package.split('v').last().and_then(|v| match v {
+            "0.2.0-rc-2023-11-10" => Some("15.0.1"),
```

Comment:
Why this mapping rather than naming the adapter with the full rc version?

Reply:
The boring answer is that the version string is long and makes me sad, and the adapter file name was named after the wasmtime version so I decided to do the mapping. It's an easy change, but I'd like to take a step back and think a bit deeper about how we want to do the automagical adapter step, but I think that can wait for a PR in the near future. How does that sound?

Reply:
`15.0.1` just seems more confusing if you don't already have deep wasmtime knowledge. :shrug: 

Certainly not a blocker.

Reply:
Yea I hear you. I can change it to your suggestion. I agree it's less confusing. This might change in the future though.